src/main/java/org/jenkinsci/plugins/authorizeproject/ProjectQueueItemAuthenticator.java

```diff
@@ -51,23 +55,37 @@
  * Authorize builds of projects configured with {@link AuthorizeProjectProperty}.
  */
 public class ProjectQueueItemAuthenticator extends QueueItemAuthenticator {
-    private final Map<String,Boolean> strategyEnabledMap;
-    
-    /**
-     * 
-     */
+
+    private final Set<String> enabledStrategies;
+    private final Set<String> disabledStrategies;
+
+    @Deprecated
+    private transient Map<String,Boolean> strategyEnabledMap;

```

Comment:
Not seeing anything that converts this old property.

Reply:
In `readResolve` it's converted to the two new properties `enabledStrategies` and `disabledStrategies`.